index.html

```diff
@@ -889,6 +884,79 @@ <h2 class='informative'>Vary</h2>
 
+    <section id='external-content'>
+      <h3>External LDP-NR Content</h3>
+      <p class='informative'>
+        Variability among client types and locations may mean that LDP-NR
+        content is addressed in ways that are external to the Fedora server but not resolvable
+        by all clients. This specification describes the use of <code>Content-Type: message/external-body</code>
+        values to signal, on POST or PUT, that the Fedora server should should not consider the
+        request entity to be the LDP-NR's content, but that a <code>Content-Type</code> value will signal a
+        name or address at which the content might be retreived. The <code>url</code>
+        and <code>local-file</code> type values motivate this specification, but a Fedora server
+        may support any type parameters per the requirements for advertisement and
+        rejection specified here.
+      </p>
+      <section id='message-external-body-accept-post'>
+        <p>
+          Fedora servers that support LDP-NR with <code>message/external-body</code>
+          MUST advertise that support in the <code>Accept-Post</code> response
+          header for each supported <code>type</code> parameter of supported <code>Content-Type</code> values.
+        </p>
+      </section>
+      <section id='message-external-body-unsupported-type'>
+        <p>
+          Fedora servers receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> with an unsupported
+          <code>type</code> parameter MUST respond with HTTP 415 UNSUPPORTED MEDIA TYPE.
+          In the case that a Fedora server does not support external LDP-NR content, all
+          <code>message/external-body</code> messages must be rejected with HTTP 415.
+        </p>
+      </section>
+      <section id='message-external-body-response-headers'>
+        <p>
+          Fedora servers receiving requests that would create or update a LDP-NR
+          with a <code>message/external-body</code> MUST NOT accept the
+          request if it cannot guarantee all of the response headers required
+          by the LDP-NR interaction model in this specification.
+        </p>
+      </section>
+      <section id='external-content-content-location'>
+        <p>
+          LDP-NR GET and HEAD responses SHOULD include a <code>Content-Location</code> header with a URI
+          representation of the location of the external content if the Fedora server is proxying
+          the content.
+        </p>
+      </section>
+      <section id='external-content-expires'>
+        <p>
+          Per [[!RFC1521]], all <code>Content-Type: message/external-body</code> values
+          MAY include an <code>expiration</code> parameter. Fedora servers receiving requests that
+          would create or update a LDP-NR with a <code>message/external-body</code> content
+          type SHOULD respect the <code>expiration</code> parameter, if present, by copying
```

Comment:
Sorry, I simply find this section hard to understand what this section means from a high level.  It suggests that a server `SHOULD` copy content if an expiration parameter is present, but it could do something else, Including (as the above thread suggests) "expire the resource", which I understand as making the LDP-NR content unavailable once the expiration date is met.  So is it consistent with the spirit of this section that one implementation might interpret `expiration` as "make this ephemeral content durable", and another implementation might interpret it as "expire the content at the appropriate time"?   It may be a stupid question, but I'd really appreciate some help from the editors to help understand how to interpret this section.